.travis.yml

```diff
@@ -0,0 +1,44 @@
+sudo: false
+
+language: java
+
+matrix:
+  fast_finish: true
+  include:
+
+  - jdk: oraclejdk8
+    env: TASK=BUILD
```

Comment:
We don't have other TASK. Consider to remove it.

Reply:
Done.